lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1678,6 +1687,9 @@ def cmd_sessions(*args)
           print_status("Starting interaction with #{session.name}...\n") unless quiet
           begin
             self.active_session = session
+
+            session.on_error_proc = log_on_timeout_error("Send timed out. Timeout currently #{session.response_timeout} seconds, you can configure this with sessions --interact <id> --timeout <value>")
```

Comment:
Looks like this code is in the wrong place, it should be in the `if session.respond_to?(:response_timeout)` guard clause above